python/cudf_polars/cudf_polars/containers/dataframe.py

```diff
@@ -83,6 +93,35 @@ def from_cudf(cls, df: cudf.DataFrame) -> Self:
             ]
         )
 
+    @classmethod
+    def from_polars(cls, df: pl.DataFrame) -> Self:
+        """
+        Create from a polars dataframe.
+
+        Parameters
+        ----------
+        df
+            Polars dataframe to convert
+
+        Returns
+        -------
+        New dataframe representing the input.
+        """
+        table = df.to_arrow()
```

Comment:
I don't know if we think it may be worth noting if a copy is made or not, otherwise this function LGTM

Reply:
I think since it's from polars, I don't think we need to note copying.